test/parallel/test-debug-port-from-cmdline.js

```diff
@@ -17,7 +17,18 @@ child.stderr.on('data', function(data) {
 
 child.on('message', function onChildMsg(message) {
   if (message.msg === 'childready') {
-    process._debugProcess(child.pid);
+    try {
+      process._debugProcess(child.pid);
+    } catch (e) {
+      if (common.isWindows && (e.message ===
+            'Not enough storage is available to process this command.' ||
+            e.message === 'Access is denied.')) {
+        child.kill();
+        console.log('Encountered IRPStackSize bug on win32, ignoring test');
```

Comment:
We have to follow the convention

    1..0 # Skipped: Encountered IRPStackSize bug on win32, ignoring test

while skipping tests, so that TAP Plugin can pick them.